rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,14 +2,14 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
 
 %{!?_root_bindir:%global _root_bindir %{_bindir}}
 
 Summary: The Foreman/Satellite maintenance tool
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.3
-Release: 1%{?dist}
+Version: 0.0.6
+Release: 2%{?dist}
```

Comment:
@ehelms What happened is, the above [test] comment didn't trigger auto tests, so I figured bumping something may trigger it. I'll set it back to 1.